gameplay/Gameplay-Guide.md

```diff
@@ -1,4 +1,4 @@
-Red Eclipse features 5 playable game modes which can be further customised through mutators. Along with the cooperative map editor and a demo viewer to playback previously recorded matches.
+Red Eclipse features 5 playable game modes which can be further customized through mutators. Along with the cooperative map editor and a demo viewer to playback previously recorded matches.
```

Comment:
We use 's' instead of 'z' in most UK spelling cases, but if you've changed them all don't worry about changing it back.